openapi.yaml

```
@@ -141,6 +179,25 @@ components:
       properties:
         asyncapi:
           $ref: '#/components/schemas/AsyncAPIDocument'
+    DiffDocument:
+      type: object
+      required:
+        - asyncapi1
+        - asyncapi2
+      properties:
+        asyncapi1:
+          $ref: '#/components/schemas/AsyncAPIDocument'
+        asyncapi2:
+          $ref: '#/components/schemas/AsyncAPIDocument'
```

I opt to send the `array` of the AsyncAPI document like:

```suggestion
      type: object
      required:
        - asyncapis
      properties:
        asyncapis:
          type: array
          minItems: 2
          maxItems: 2
          items:
            $ref: '#/components/schemas/AsyncAPIDocument'
```

Also `diff` package has parameters that you can pass to change behaviour of the diff library: https://github.com/asyncapi/diff/blob/master/src/types.ts#L55 Please propagate these types in the `openapi.yaml` and then in the controller :)

@BOLT04 What do you think? :)